README.md

```diff
@@ -4,22 +4,26 @@
 
 ## Table of Contents
 
-1. [Naming](#naming)
+1. [Eslint](#eslint)
+2. [Naming](#naming)
    1. [Casing](#casing)
    2. [File names](#file-names)
-2. [Coding](#coding)
-3. [Formatting](#formatting)
-4. [Comments](#comments)
-5. [TypeScript](#typescript)
-6. [GIT](#git)
+3. [Coding](#coding)
```

Comment:
Tip, just use `1.` everywhere, when rendered to HTML it will do the proper increments.

Reply:
There is a script handling this. So this is not changed manually 😁